extensions/cornerstone/src/commandsModule.ts

```diff
@@ -612,6 +613,27 @@ function commandsModule({ servicesManager, commandsManager }) {
       );
       toolGroup.setToolEnabled(ReferenceLinesTool.toolName);
     },
+
+    /**
+     * Toggle Image Overlays
+     *
+     * @param param0
+     */
+    toggleImageOverlay: ({ toggledState }) => {
+      toggledState = !toggledState; // revert the state because unlike other toggle tools, this starts as enabled
```

Comment:
You can just add a comment to line 633 and reverse the two conditions to set enabled/disabled.  Then, add the comment right over that to indicate that the "on" toggled stage means pixel overlay hidden.